modules/monitoring-enabling-an-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,103 @@
+//
+// * monitoring/configuring-alerts-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-enabling-an-alertmanager-instance-for-user-defined-projects_{context}"]
+= Enabling an Alertmanager instance for user-defined projects
+
+A cluster administrator can enable a dedicated instance of Alertmanager to use with user-defined projects in a cluster.
+Doing so enables developers and other non-administrator users of user-defined projects to set up individual alerting configurations to suit their needs without the help of an administrator.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have enabled monitoring for user-defined projects.
+* You have enabled alert routing for user-defined projects.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `user-workload-monitoring-config` `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap user-workload-monitoring-config
+----
++
+. Add `alertmanager: true` under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
+data:
+  config.yaml: |
+    alertmanager: true <1>
+----
+<1> Set the `alertmanager` key to `true` to enable a dedicated instance of the Alertmanager for user-defined projects in a cluster. Set the value to `false` or omit the key entirely to disable the Alertmanager for user-defined projects.
++
+. Save the file to apply the changes. The dedicated instance of Alertmanager for user-defined projects starts automatically.
+
+.Verification
+
+. Verify that the `alertmanager-user-workload` pods have been created:
++
+[source,terminal]
+----
+# oc -n openshift-user-workload-monitoring get pod
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME                                   READY   STATUS    RESTARTS        AGE
+alertmanager-user-workload-0           6/6     Running   0               6m27s
+alertmanager-user-workload-1           6/6     Running   0               6m27s
+prometheus-operator-549fc76b94-s9kgp   2/2     Running   0               71m
+prometheus-user-workload-0             6/6     Running   0               71m
+prometheus-user-workload-1             6/6     Running   0               71m
+thanos-ruler-user-workload-0           3/3     Running   1 (6m20s ago)   6m25s
+thanos-ruler-user-workload-1           3/3     Running   1 (6m25s ago)   6m29s
+----
++
+. Verify that the Alertmanager instance has started:
++
+[source,terminal]
+----
+#  oc -n openshift-user-workload-monitoring get alertmanager
+----
++
+.Example output
++
+[source,terminal]
+----
+NAME            VERSION   REPLICAS   AGE
+user-workload   0.24.0    2          100s
+----
++
+. Verify that the Alertmanager service has started:
++
+[source,terminal]
+----
+#  oc -n openshift-user-workload-monitoring get svc
```

Comment:
remove additional space after `#`
should be
`# oc -n openshift-user-workload-monitoring get svc`